### PR TITLE
polished code

### PR DESCRIPTION
**Pull request by sveneberth**
- removed unnecessary debugs
- fixed several typos 
- renamed id --> key
- added utf-8 encoding in top of all files
- corrected datatypes in docstrings
- changed param syntax of several docstrings
- improved readability of some large dicts and bones

### CODE REVIEW

File: bones/bone.py [hunk 1]
```diff
@@ -418,7 +418,7 @@ def postDeletedHandler(self, skel, boneName, key):
 			:param boneName: Name of this bone
 			:type boneName: String
 			:param key: The old Database Key of hte entity we've deleted
```
Comment (phorward) [inline bones/bone.py]: hte=the

File: modules/page.py [hunk 2]
```diff
@@ -13,17 +9,17 @@ class pageSkel( HierarchySkel ):
 	descr = textBone( descr="Content", required=True, searchable=True )
 
 class Page( Hierarchy ):
-	adminInfo = {	"name": "Sites", #Name of this modul, as shown in ViUR Admin (will be translated at runtime)
+	adminInfo = {	"name": "Sites", #Name of this module, as shown in ViUR Admin (will be translated at runtime)
 				"handler": "hierarchy",  #Which handler to invoke
-				"icon": "icons/modules/hierarchy.svg", #Icon for this modul
-				"formatstring": "$(name)", 
-				"filters" : {
```
Comment (phorward) [inline modules/page.py]: What is this for a piece of code? I know filters as a dict of filter criteria, but not with a key None containing a dict. 
I think this is a very obsolete definition of a views entry... can you please convert this to a current implementation, according to https://github.com/viur-framework/vi/wiki/adminInfo-in-List-modules#views ?

**Comment by skoegl** [comment]
Hey there, just one note: I've still seeing several :type foo: String annotations. But we only have str as type in python. That could confuse doc readers, as if we would have some own string as class implementation.

**Comment by sveneberth** [comment]
> Hey there, just one note: I am still seeing several :type foo: String annotations. But we only have str as type in python. That could confuse doc readers, as if we would have some own string as class implementation.

good mention!

**Pull request merged**